style.css

```diff
@@ -414,3 +415,209 @@
     background-color: #6070ff;
   }
 }
+
+@media screen and (min-width: 768px) {
+  #header {
+    height: 90px;
+    width: 100%;
+    padding-bottom: 40px;
+  }
+
+  .logo {
+    padding-left: 155px;
+    padding-top: 10px;
+  }
+
+  .nav {
+    --i1: scale(0);
+    --i2: scale(0);
+  }
+
+  .nav:has(.side-bar:target) {
+    --i1: scale(0);
+    --i2: scale(0);
+    --clip: inset(0 0 0 0);
+  }
+
+  .nav-container {
+    width: 100%;
+    margin: 0 auto;
+    display: flex;
+  }
+
+  .nav-menu {
+    display: none;
+  }
+
+  .nav-menu-second {
+    display: none;
+  }
+
+  .side-bar {
+    position: relative;
+    width: 100%;
+    background-color: #fff;
+    clip-path: none;
+    transition: none;
+    display: flex;
+    margin-right: 155px;
+  }
+
+  .side-bar li {
+    padding-right: 32px;
+    display: flex;
+    padding-bottom: 10px;
+  }
+
+  .side-bar li a {
+    color: rgba(52, 69, 99, 1);
+    font-size: 15px;
+  }
+
+  .social-container li a {
+    font-size: 16px;
+  }
+
+  .side-bar li a:hover {
+    color: rgba(52, 69, 99, 0.638);
+  }
+
+  .section-p1 {
+    background-image: url('img/Header-bg1.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top center;
+    height: 590px;
+    border-bottom: 1px solid rgba(223, 225, 230, 1);
+  }
+
+  .section-p1 .div1 {
+    margin-top: 90px;
+    margin-left: 360px;
+  }
+
+  .div1 h2 {
+    font-size: 48px;
+    width: 30%;
+    line-height: 60px;
+  }
+
+  .div1 p {
+    font-size: 20px;
+    width: 50%;
+  }
+
+  .section-p1 .div2 {
+    margin-left: 90px;
+    margin-left: 360px;
+  }
+
+  .section-p2 {
+    width: 100%;
+  }
+
+  .card {
+    width: 1156px;
+    height: 496px;
+    display: flex;
+    flex-direction: row;
+    border: 1px solid rgba(223, 225, 230, 1);
+  }
+
+  .card-multi {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .card .img-project {
+    width: 100%;
+    height: 100%;
+    padding: 24px;
+  }
+
+  .description-project h3 {
+    font-size: 40px;
+    padding-bottom: 20px;
+  }
+
+  .multi {
+    padding-left: 50px;
+  }
+
+  .multi h3 {
+    width: 60%;
+    line-height: 50px;
+    padding-right: 10px;
+  }
+
+  .frame {
+    justify-content: space-evenly;
+  }
+
+  .section-p3 {
+    flex-direction: row;
+    height: 668px;
+    align-items: center;
+  }
```

Comment:
- [ ] Nice work implementing the "About Myself" section :clap: However make the left content have 40% and the right content 60% of the page's width to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-04 10-21-51](https://user-images.githubusercontent.com/91142432/236187519-6d44860a-086d-48eb-98b1-44a77d7d5ee1.png)
### Figma design :point_down: 
![Screenshot from 2023-04-05 06-38-27](https://user-images.githubusercontent.com/91142432/236187623-36165541-ddac-43d7-a97b-a82508fb5315.png)